product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md

```diff
@@ -15,6 +17,14 @@ Das _fiskaltrust.Middleware.SCU.DE.Epson_-Package ermöglicht die Verbindung der
 | _deviceid_ | Die für die TSE spezifische Geräte-ID. Falls nicht angegeben, wird der Defaultwert verwendet. | ja |
 | _timeout_ | Timeout-Wert für die Kommunikation mit der TSE in Millisekunden. Falls nicht angegeben, wird der Default-Wert von 60 Sekunden verwendet. | ja |
 
+## Konfiguration
+
+![scu-de-epson-config](../media/scu-de-epson-config.png)
+
+- **Host:** The DNS or IP Address of the Epson Server, Epson Printer or local webservice
```

Comment:
If the rest of the article is in german we should also use german here :)

Reply:
Actually we could use the old parameters and just change tseurl and tseport

Reply:
jepp